code/datums/mood_events/generic_negative_events.dm

```diff
@@ -353,3 +353,7 @@
 	description = "<span class='boldwarning'>I can hardly breathe with all this HERESY going on!</span>\n"
 	mood_change = -5
 	timeout = 5 MINUTES
+
+/datum/mood_event/gas_rash
+	description =  "<span class='boldwarning'>My skin is all red! I need to remove it!</span>\n"
```

Comment:
Remove it? what are you trying to get across?

Reply:
ehh didn't know what to write there, is more like an excuse for the mood debuff, i'll try to come up with something better